src/test/deploy/functions/params.spec.ts

```diff
@@ -0,0 +1,134 @@
+import * as chai from "chai";
+import * as sinon from "sinon";
+chai.use(require("chai-as-promised"));
+const expect = chai.expect;
+const prompt = require("../../../prompt");
```

Comment:
1. Why are you using requires instead of imports?
2. imports should have one block of external imports followed by a newline followed by a block of internal imports.